substrate/frame/revive/fixtures/riscv32emac-unknown-none-polkavm.json

```diff
@@ -0,0 +1,26 @@
+{
+  "arch": "riscv32",
+  "cpu": "generic-rv32",
+  "crt-objects-fallback": "false",
+  "data-layout": "e-m:e-p:32:32-i64:64-n32-S32",
+  "eh-frame-header": false,
+  "emit-debug-gdb-scripts": false,
+  "features": "+e,+m,+a,+c,+lui-addi-fusion,+fast-unaligned-access,+xtheadcondmov",
```

Comment:
```suggestion
  "features": "+e,+m,+a,+c,+lui-addi-fusion,+unaligned-scalar-mem,+xtheadcondmov",
```

Reply:
Why? I copied the file from here:
https://github.com/paritytech/polkavm/blob/master/guest-programs/riscv32emac-unknown-none-polkavm.json

Reply:
`+fast-unaligned-access` isn't a feature for the target and it won't have any effect. `+fast-unaligned-access` seems to be an older artifact. 

https://github.com/llvm/llvm-project/blob/main/llvm/lib/Target/RISCV/RISCVFeatures.td#L1358-L1361

Reply:
Ah okay it still called ` +fast-unaligned-access` in LLVM 18.X. Rust `1.82.0` (latest stable) however:
```
rustc -vV
rustc 1.82.0 (f6e511eec 2024-10-15)
binary: rustc
commit-hash: f6e511eec7342f59a25f7c0534f1dbea00d01b14
commit-date: 2024-10-15
host: x86_64-unknown-linux-gnu
release: 1.82.0
LLVM version: 19.1.1
```

What toolchain do we expect?

Reply:
I guess the yolo fix would be to set both flags. They are just warned about and ignored if not supported